doc/manual/src/command-ref/env-common.md

```diff
@@ -86,6 +86,11 @@ Most Nix commands interpret the following environment variables:
     from (defaults to the XDG spec locations). The variable is treated
     as a list separated by the `:` token.
 
+  - `NIX_OPTIONS`
```

Comment:
This line needs two trailing spaces to ensure correct formatting as a definition list.